.github/workflows/ios_main.yml

```diff
@@ -0,0 +1,25 @@
+name: build
+on: [push]
+
+jobs:
+  build:
+    name: build
+    runs-on: macos-latest
+    strategy:
+        matrix:
+          destination: ['platform=iOS Simulator,OS=13.1,name=iPhone 11']
```

Comment:
does github actions prove all the OS versions all the time? or do we have to update this field every time a new version is released? is there a way to use the latest one automatically?

Reply:
unfortunately, I didn't find any way how to use OS=latest, or something like that, there is an option to not use OS at all, but one still has to specify the phone name, which doesn't solve this issue anyway